### ISSUE TITLE
Additional data needed?

### ISSUE THREAD

**Issue by sterlingbaldwin** [ACME-Climate/a-prime#13]
Running a-prime again, and getting the following error:
```
MPAS-Analysis submodule:  30b394a73fc47f5d8f25f9c1e5b5685ed529cc74 python/MPAS-Analysis (v0.5)
Traceback (most recent call last):
  File "python/MPAS-Analysis/run_mpas_analysis.py", line 321, in determine_analyses_to_generate
    analysisTask.setup_and_check()
  File "/p/cscratch/acme/data/a-prime/python/MPAS-Analysis/mpas_analysis/ocean/meridional_heat_transport.py", line 107, in setup_and_check
    'meridionalHeatTransportOutput')
  File "/p/cscratch/acme/data/a-prime/python/MPAS-Analysis/mpas_analysis/shared/io/namelist_streams_interface.py", line 384, in readpath
    path, self.fname, streamName))
ValueError: Path /p/cscratch/acme/baldwin32/20171101_yes_sta/output/tmp/aprime/0051-0060/20170915.beta2.A_WCYCL1850S.ne30_oECv3_ICG.edison/run/mpaso.hist.am.meridionalHeatTransport.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9].nc in streams file /p/cscratch/acme/baldwin32/20171101_yes_sta/output/tmp/aprime/0051-0060/20170915.beta2.A_WCYCL1850S.ne30_oECv3_ICG.edison/run/streams.ocean for 'meridionalHeatTransportOutput' not found.
ERROR: analysis module meridionalHeatTransport failed during check and will not be run
```

Is there additional data needed that isnt listed in the documentation? For example  mpaso.hist.am.meridionalHeatTransport.*.nc

Im creating my own directories and only moving in the data which the documentation says I need, Im not running on the actual E3SM model output directory.

**Comment by xylar** [comment]
@sterlingbaldwin, you are correct.  A single `mpaso.hist.am.meridionalHeatTransport.*.nc` file is needed (any one) in order to perform the analysis of meridional heat transport.  @milenaveneziani, that should be added to the documentation in `run_aprime.bash`.

**Comment by milenaveneziani** [comment]
hmm, I could have sworn I had added this piece of information in the header of run_aprime.. It must have gotten lost somehow. I will add it in the next update.
sorry about that.

**Comment by sterlingbaldwin** [comment]
Thanks for the help!

**Comment by milenaveneziani** [comment]
Addressed by #24

**Comment by sterlingbaldwin** [comment]
Thanks for addressing this, I think [this](https://github.com/ACME-Climate/a-prime/pull/24/files#diff-028252433d47ebf34e5a0285e717d4deR57) was my issue
